Source/WebCore/css/CSSValueKeywords.in

```diff
@@ -732,6 +732,10 @@ column-reverse
 // wrap
 wrap-reverse
 
+// CSS_PROP_MARGIN_TRIM
+block-start
+block-end
```

Comment:
For consistency with other props:
```suggestion
// block
block-start
block-end
// inline
// inline-start
// inline-end
```

Reply:
Changed (inline-start and inline-end were already defined for other properties it seems so I moved them)

Reply:
inline-start and inline-end are also valid values for `float` and `clear`, so they shouldn't be removed from the section above.

You should include them again here but commented out.

Reply:
Done